garak/cli.py

```diff
@@ -247,6 +253,11 @@ def main(arguments=None) -> None:
         action="store_true",
         help="Launch garak in interactive.py mode",
     )
+    parser.add_argument(
+        "--detector_only",
+        action="store_true",
+        help="run detector on jsonl report"
+    )
```

Comment:
I'm not exactly sure if `continue` or `rescore` has been implemented yet (or maybe in some other branch?). But I agree with creating `harness_options` instead of exposing a lot of unnecessary higher-level options. I have incorporated the idea of `harness_options` in the new changes.